lecture#17/README.md

```diff
@@ -0,0 +1,22 @@
+# Web Scraping in Python
+
+- Speaker: [Gavrilita Mihail](https://www.facebook.com/mihail.gavrilita.7)
+- Date: 14.05.2019
+- Facebook event: https://www.facebook.com/events/418819118899017
+
+## Long Story Short
+
+Let me start with a pretty bold statement: 
+You can find pretty much anything on the Internet. 
+But there's a catch: 
+It's not always in one place. 
+The data you need can be located in different sources and different websites, scattered on pages or available for download in one file. 
+How to collect this data efficiently?
+
+You could hire a team of highly trained copy-pasting monkeys (they need bananas..) or
+you could write a small script in Python, using a [framework](https://docs.scrapy.org/en/latest/intro/overview.html) which makes automatic web scraping a breeze.
+
+In this repository you can find the slides used for this lecture (useful links on the last slide),
+as well as a small project that we used to scrape the whole [menu](https://www.andys.md/en/pages/menu/) of Andy's Pizza restaurant.
+It took us less than 2 minutes :simple_smile:
```

Comment:
Please check https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md to selected an emoji supported by Github.